src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString.split(';');
+  const resultConvert = {};
+  let properties = [];
+  for (let i = 0; i < array.length; i++) {
+    array[i] = array[i].trim();
+    properties = array[i].split(': ');
```

Comment:
properties not the best name, because variable contains 1 css property, `nameValuePair`, `style`
Or use destruction and create 2 variables instead